app/controllers/trips_controller.rb

```diff
@@ -3,9 +3,10 @@ class TripsController < ApplicationController
 
   def show
     @trip = TripFacade.trip_by_id(@user.google_id, params[:id])
-
+    @lat_long = [39.7391, -104.9849]
```

Comment:
We don't wanna hardcode this - gonna have to call lat long services to get the users current location? or maybe latlong of the starting address?